Aptoide/aptoide-client-v8
# [ASV-1828] Fix ad mob problems

**What does this PR do?**

   This PR aims at increasing the AdMob sdk dependency version.

**Database changed?**

 No

**Where should the reviewer start?**


**How should this be manually tested?**

  Open the app, confirm that you are seeing ads. Also since AdMob is a certified network, it is initialized by MoPub, so you can also check that by adding the filter MoPub in the logcat you can see the initialization logs, where AdMob should be included.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1828](https://aptoide.atlassian.net/browse/ASV-1828)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 

AdMob key which you should already have.

**Code Review Checklist**
